Sources/PrimerSDK/Classes/Core/BIN Data/CardValidationService.swift

```diff
@@ -71,7 +74,7 @@ class DefaultCardValidationService: CardValidationService, LogReporter {
                                                   forCardState: cardState)
         }.catch { error in
             // JN TODO: CHKT-1772 - send event
-            self.logger.error(message: "Error occurred while remotely validation '\(cardState.cardNumber)': \(error.localizedDescription)")
+            self.logger.error(message: "Error occurred while remote card validation: \(error.localizedDescription)")
```

Comment:
Should this be a warning?

Reply:
Good question. I think it's an error because we can't display the network and, potentially (TBC), it means that a payment can't be completed.

What are your thoughts?

(Also, the grammar is terrible here - will update)

Reply:
Yeah if we won't fallback to local validation then this should be an error. If we are having a fallback it should be a warning as we are not breaking the flow.

Reply:
I'm 50/50 because this specific thing doesn't break the flow, but you won't be able to continue if we can't get the only supported network (e.g. CB is configured, but the API fails, so we can't complete payment)